packages/fsweb/package.json

```diff
@@ -8,7 +8,9 @@
     "build": "webpack --mode production",
     "build:size": "webpack --json | webpack-bundle-size-analyzer",
     "start": "webpack-dev-server -d --config ./webpack.config.js --mode development --inline --hot",
-    "start-prod": "webpack-dev-server -d --config ./webpack.config.js --mode production"
+    "start-prod": "webpack-dev-server -d --config ./webpack.config.js --mode production",
+    "server-dev": "node ./proxy/server.dev.js",
+    "server-prod": "node ./proxy/server.prod.js"
```

Comment:
These commands should also be added to pirateship, fstestproject, and `ship:` versions added to the base package. The files also need to be updated so that they will point at the env pile in the project.

Reply:
@Cauldrath done